### ISSUE TITLE
[V2]The preview window pops up unexpectedly

### ISSUE THREAD

**Issue by o000u** [p0deje/Maccy#839]
### Before Submitting Your Bug Report

- [ ] I have verified that there isn't already an issue reporting the same bug to prevent duplication.
- [ ] I have seen the [FAQ](https://github.com/p0deje/Maccy?tab=readme-ov-file#faq).

### Maccy Version (see 'About' window)

2.0.0.beta.10

### macOS Version

14.6

### Maccy Settings

```Shell
{
    "KeyboardShortcuts_delete" = "{\\"carbonModifiers\\":2048,\\"carbonKeyCode\\":51}";
    "KeyboardShortcuts_pin" = "{\\"carbonModifiers\\":2048,\\"carbonKeyCode\\":35}";
    "KeyboardShortcuts_popup" = "{\\"carbonKeyCode\\":9,\\"carbonModifiers\\":4096}";
    "NSStatusItem Visible Item-0" = 0;
    "NSWindow Frame com.sindresorhus.Settings.FrameAutosaveName" = "705 408 510 473 0 0 1920 1080 ";
    SUEnableAutomaticChecks = 0;
    SUHasLaunchedBefore = 1;
    avoidTakingFocus = 1;
    clearOnQuit = 1;
    enabledPasteboardTypes =     (
        "public.tiff",
        "public.png",
        "public.rtf",
        "public.html",
        "public.utf8-plain-text"
    );
    maxMenuItemLength = 50;
    migrations =     {
        "2024-07-01-version-2" = 1;
    };
    pasteByDefault = 1;
    pinTo = bottom;
    previewDelay = 2000;
    showFooter = 0;
    showInStatusBar = 0;
    showSearch = 1;
    showTitle = 0;
    windowSize = "[360,408]";
}
```

### Description

I set the preview delay to 2000, but sometimes, when you activate the clipboard panel, the preview window pops up immediately without delay

### Steps to Reproduce
1. Use the shortcut key to display the clipboard panel first and then hide it immediately
2. After a period of time (preferably exceeding the preview delay time you set), use the shortcut key to display the clipboard panel again. At this time, the preview window should appear immediately without delay

_No response_

**Comment by weisJ** [comment]
Probably due to this line: https://github.com/p0deje/Maccy/blob/dd4e43dd7c95c0a1769f0d3df15fba4877880198/Maccy/Observables/HistoryItemDecorator.swift#L27

**Comment by p0deje** [comment]
Should be fixed in 2.0.0.beta.11

**Comment by o000u** [comment]
> Should be fixed in 2.0.0.beta.11

Hi @p0deje ,I'm glad you fixed this bug in 2.0.0.beta.11, but it still seems that the preview window pops up immediately. 

Steps to Reproduce
1.Click any entry in the clipboard panel
2.After a period of time (preferably exceeding the preview delay time you set), use the shortcut key to display the clipboard panel again. At this time, the preview window should appear immediately without delay

**Comment by o000u** [comment]
> > Should be fixed in 2.0.0.beta.11
> 
> Hi @p0deje ,I'm glad you fixed this bug in 2.0.0.beta.11, but it still seems that the preview window pops up immediately.
> 
> Steps to Reproduce 1.Click any entry in the clipboard panel 2.After a period of time (preferably exceeding the preview delay time you set), use the shortcut key to display the clipboard panel again. At this time, the preview window should appear immediately without delay

WOW!～～～
It has been fixed in 2.0.0.beta.12!!, thank you for your efficient and excellent work!!! @p0deje 
Love Maccy forever❤️!!
